cluster.go

```diff
@@ -393,7 +412,11 @@ func (c *Cluster) traceTopoChanged(prevTopo ClusterTopo, newTopo ClusterTopo) {
 
 // while this method is normally deduplicated by the Sync method's use of
 // dedupe it is perfectly thread-safe on its own and can be used whenever.
-func (c *Cluster) sync(p Client) error {
+func (c *Cluster) sync(p Client, args ...bool) error {
```

Comment:
`sync` should take in just one boolean to indicate if errors should be silenced, this `...bool` pattern is not idiomatic in go.